pkg/inventory/model/table.go

```diff
@@ -305,38 +294,32 @@ func (t Table) Insert(model interface{}) error {
 func (t Table) Update(model interface{}) error {
 	fields, err := t.Fields(model)
 	if err != nil {
-		Log.Trace(err)
-		return err
+		return liberr.Wrap(err)
 	}
 	stmt, err := t.updateSQL(t.Name(model), fields)
 	if err != nil {
-		Log.Trace(err)
-		return err
+		return liberr.Wrap(err)
 	}
 	params := t.Params(fields)
 	r, err := t.Db.Exec(stmt, params...)
 	if err != nil {
-		Log.Trace(err)
-		return err
+		return liberr.Wrap(err)
 	}
 	nRows, err := r.RowsAffected()
 	if err != nil {
-		Log.Trace(err)
-		return err
+		return liberr.Wrap(err)
 	}
 	if nRows == 0 {
 		if t.Get(model) == nil {
-			return Conflict
+			return liberr.Wrap(Conflict)
 		} else {
-			return NotFound
+			return liberr.Wrap(sql.ErrNoRows)
```

Comment:
Is there a reason that this instance of `sql.ErrNoRows` isn't called `NotFound`?

Reply:
Oh, not really.  Meant to just wrap the NotFound.
Fixed.